tests/disaster-recovery/sc_arbiter/test_netsplit.py

```diff
@@ -0,0 +1,450 @@
+            else:
+                logger.info(f"Read success: {t_time}")
+            time_var = time_var + timedelta(minutes=1)
+        if pause_count > 5:
+            paused = True
+            break
+    return paused
+
+
+def check_for_write_pause(logwriter_pod, log_files, start_time, end_time):
+    paused = False
+    for file_name in log_files:
+        pause_count = 0
+        file_log = logwriter_pod.exec_sh_cmd_on_pod(command=f"cat {file_name}")
+        time_var = start_time
+        logger.info(f"Current file: {file_name}")
+        while time_var <= (end_time + timedelta(minutes=1)):
+            t_time = time_var.strftime("%H:%M")
+            if f"T{t_time}" not in file_log:
+                pause_count += 1
+                logger.info(f"Write pause: {t_time}")
+            else:
+                logger.info(f"Write success: {t_time}")
+            time_var = time_var + timedelta(minutes=1)
+        if pause_count > 5:
+            paused = True
+            break
+    return paused
+
+
+class TestNetSplit:
+    @pytest.fixture()
+    def init_sanity(self):
+        self.sanity_helpers = Sanity()
+
+    @pytest.mark.parametrize(
+        argnames="zones, duration",
+        argvalues=[
+            pytest.param(constants.NETSPLIT_DATA_1_DATA_2, 15),
+            pytest.param(constants.NETSPLIT_ARBITER_DATA_1, 15),
+            pytest.param(constants.NETSPLIT_ARBITER_DATA_1_AND_ARBITER_DATA_2, 15),
+            pytest.param(constants.NETSPLIT_ARBITER_DATA_1_AND_DATA_1_DATA_2, 15),
+        ],
+        ids=[
+            "Data-1-Data-2",
+            "Arbiter-Data-1",
+            "Arbiter-Data-1-and-Arbiter-Data-2",
+            "Arbiter-Data-1-and-Data-1-Data-2",
+        ],
+    )
+    def test_netsplit_cephfs(
+        self,
+        setup_logwriter_cephfs_workload_factory,
+        logreader_workload_factory,
+        nodes,
+        zones,
+        duration,
+        init_sanity,
+    ):
+        """
+        This test will test the netsplit scenarios when active-active CephFS workload
+        is running.
+        Steps:
+            1) Run both the logwriter and logreader CephFS workload using single RWX volume
+            2) Induce the network split
+            3) Make sure logreader job pods have Completed state.
+               Check if there is any write or read pause. Fail only when neccessary.
+            4) For bc/ab-bc netsplit cases, it is expected for logreader/logwriter pods to go CLBO
+               Make sure the above pods run fine after the nodes are restarted
+            5) Delete the old logreader job and create new logreader job to verify the data corruption
+            6) Make sure there is no data loss
+            7) Do a complete cluster sanity and make sure there is no issue post recovery
+
+        """
+
+        # run cephfs workload for both logwriter and logreader
+        logwriter_workload, logreader_workload = setup_logwriter_cephfs_workload_factory
+        time.sleep(60)
```

Comment:
as discussed, instead of adding sleep here add a logic to retry the failed command